Dockerfile.spdk

```diff
@@ -0,0 +1,107 @@
+# syntax = docker/dockerfile:1.4
+ARG OS_RELEASE=9
+FROM registry.access.redhat.com/ubi$OS_RELEASE/ubi:latest as prebase
+ARG OS_RELEASE
+
+# Speed up dnf
```

Comment:
"Speed up dnf" is not really clear to me - is it simply the `tsflags=nodocs`? 

If we switch to `ubi9-minimal` and `microdnf`, it's worth re-evaluating if each of these settings is still required.